### DISCUSSION TITLE
No Fleet Hosts are accessible - Check connectivity to Fleet Host

### DISCUSSION THREAD

**Discussion by blockanz** [Security-Onion-Solutions/securityonion#11205] [2.4]
### Version

2.4.10

### Installation Method

Security Onion ISO image

### Description

configuration

### Installation Type

Standalone

### Location

on-prem with Internet access

### Hardware Specs

Exceeds minimum requirements

### Network Traffic Collection

tap

### Network Traffic Speeds

Less than 1Gbps

### Status

Yes, all services on all nodes are running OK

### Salt Status

No, there are no failures

### Logs

No, there are no additional clues

### Detail

I have successfully installed standalone using the ISO. All services showing as running.

When configuring I specified IP address for connectivity.

Have downloaded windows agent and trying to install. I get the following error in the log on the windows host:
No Fleet Hosts are accessible - Check connectivity to Fleet Host

Checked netstat and I can see port 8220 open by docker-proxy so looks like it should be accessible, but nothing connecting.

Verified that local host firewall has inbound/outbound port 8220 open also.

Any other pointers for me to look at?

### Guidelines

- [X] I have read the above statement and can confirm my post is relevant to Security Onion 2.4.

**Comment by defensivedepth** [comment]
Can you please post the full install log for the Elastic Agent, sanitizing as needed? 

Also, is there any kind of access control device /  firewall between the Windows system and Security Onion?

**Comment by Lestat21** [comment]


**Reply by defensivedepth** [reply]
@Lestat21 Please start a new Discussion
